### DISCUSSION TITLE
Will pro pricing plan allow for downloads of more than 10,000 records?

### DISCUSSION THREAD

**Discussion by gfielding** [AntlerVC/firetable#1075] [Support / Q&A] [answered]
We've got over 40,000 users and growing that we need to be able to export, but we get this message: No filters or sorting will be applied on the exported data. Limited to 10,000 rows.

**Comment by harinij** [comment]
Hi @gfielding yes we are working on adding this to the Pro features - to allow downloading of tables with >10K. The downloaded file instead of being available instantly via the browser, will be emailed to the user once it's ready - as it takes a few minutes to process.

**Comment by gfielding** [comment]
Thank you @harinij 
Any idea when that might be available? if I upgrade to Pro today, will i be able to download 50,000+ records?

**Reply by BeeBombshell** [reply]
@gfielding, this feature is currently being worked on and would be out within a few weeks. We'd notify you once it's ready. You could check out our roadmap for more upcoming features here: https://roadmap.rowy.io/roadmap.

**Comment by harinij** [comment, marked as answer]
Hi @gfielding this feature is now available in the PRO plan and you can download any number of rows >10K with export now.
